internal/exec/util/user_group_lookup_test.go

```diff
@@ -63,7 +63,10 @@ func TestUserLookup(t *testing.T) {
 
 	// perform a user lookup to ensure libnss_files.so is loaded
 	// note this assumes /etc/nsswitch.conf invokes files.
-	user.Lookup("root")
+	_, err := user.Lookup("root")
+	if err != nil {
+		t.Fatalf("user lookup failed: %v", err)
```

Comment:
Hmm, I wonder if we actually want to be failing here. I wonder if we should maybe update the error message to directly point out that's what we're checking for?